src/main/java/com/spotifyapi/model/SpotifyArtist.java

```diff
@@ -0,0 +1,25 @@
+package com.spotifyapi.model;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "spotify_artist")
+@AllArgsConstructor
+@NoArgsConstructor
+public class SpotifyArtist {
+
+    @Id
+    private String id;
```

Comment:
What about generated strategy?

Reply:
its my plan to save primary id on each artist which is set spotify platform